AI_Engine_Development/Design_Tutorials/06-fft2d_AIEvsHLS/AIE/design/aie_src/graph.h

```diff
@@ -39,7 +39,7 @@
 ////////////////////////////////////////////////////////
 // FFT_2D Datatype related Macros
 // datatypes can be, cint16 (Default) or cfloat...
-#if FFT_2D_DT == cint16
+#if FFT_2D_DT == 0 
```

Comment:
The comment above says "FFT_2D Datatype related Macros datatypes can be, cint16 (Default) or cfloat".
Since it's now changed to 0 and 1, could you update its definition in comments?

Reply:
The condition made was if FFT_2D_DT = 0, then FFT_2D_DT_DIR=cint16 and vice versa for cfloat. This condition was applied in all the source files ,So I added the same in Makefile too.